### ISSUE TITLE
`/_offline` Page Does Not Work with Next.js `pageExtensions` Config Option

### ISSUE THREAD

**Issue by derek-fong** [shadowwalker/next-pwa#261]
# Summary

`/_offline` page does not work [as expected](https://github.com/shadowwalker/next-pwa#offline-fallbacks) when Next.js app has [custom page extensions](https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions) configurated (e.g. `_offline.page.tsx`). 

# Versions

- `next-pwa`: 5.2.24
- `next`: 11.0.1

# How To Reproduce

1. Clone [offline-fallback-v2 example](https://github.com/shadowwalker/next-pwa/tree/master/examples/offline-fallback-v2) from this repo. 
2. `npm install`, `npm run build` and `npm start`. Launch the app from Google Chrome and simulate offline - `/_offline` page is shown as expected. 
3. Modify `next.config.js` to include the `pageExtensions` option as described in [Next.js docs](https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions): 

```js
const withPWA = require('next-pwa')

module.exports = withPWA({
  pageExtensions: ['page.js', 'page.jsx'],
  pwa: {
    dest: 'public',
    fallbacks: {
      image: '/static/images/fallback.png',
    }
  }
})
```

4. Rename pages with `*.page.js` extensions (i.e. `_document.page.js`, `_offline.page.js`, `_index.page.js`). 

5. `npm run build` and `npm start` the app and launch the app from Google Chrome to simulate offline again. 

6. `/_offline` page will not show. The screenshot attached below is shown instead. 

# Expected Behaviors

The app should fallback to `/_offline` page when the app is offline, as described in [README - Offline Fallbacks](https://github.com/shadowwalker/next-pwa#offline-fallbacks). 

# Screenshots

![Capture](https://user-images.githubusercontent.com/10300650/128969625-5b32d8ff-b954-4130-98f3-3a88fedbe070.JPG)

# Additional Context

- Sample app: https://next-template-derekfong.vercel.app
- Next.js Docs - Custom Page Extensions: https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions

**Comment by shadowwalker** [comment]
Didn't see that use case before, I guess I could support this quickly.

**Comment by shadowwalker** [comment]
Supported in version `5.2.25`: https://github.com/shadowwalker/next-pwa/commit/67e4d58b31d52180b5eb603cab839705851a59d4

**Comment by derek-fong** [comment]
Hi @shadowwalker thanks for working on this. 

I tried to run `npm run build` after upgrading `next-pwa` to `5.2.25` and I am getting this error message: 

```
ReferenceError: pageExtensions is not defined
    at getFallbackEnvs (C:\Users\user\Desktop\sample-app\web\node_modules\next-pwa\build-fallback-worker.js:23:22)
    at buildFallbackWorker (C:\Users\user\Desktop\sample-app\web\node_modules\next-pwa\build-fallback-worker.js:58:16)
    at Object.webpack (C:\Users\user\Desktop\sample-app\web\node_modules\next-pwa\index.js:169:22)
    at getBaseWebpackConfig (C:\Users\user\Desktop\sample-app\web\node_modules\next\dist\build\webpack-config.js:162:454)
    at async Promise.all (index 0)
    at async Span.traceAsyncFn (C:\Users\user\Desktop\sample-app\web\node_modules\next\dist\telemetry\trace\trace.js:6:584)
    at async C:\Users\user\Desktop\sample-app\web\node_modules\next\dist\build\index.js:11:947
    at async Span.traceAsyncFn (C:\Users\user\Desktop\sample-app\web\node_modules\next\dist\telemetry\trace\trace.js:6:584)
```

Any idea...?

**Comment by shadowwalker** [comment]
Fixed in `5.3.1`

**Comment by derek-fong** [comment]
Thanks @shadowwalker it's all working now 👍🏻
